### PR TITLE
boards: xtensa: m5stack_core2: swap-xy touch input

### PR DESCRIPTION
**Pull request by szczys**
When running the `samples/subsys/display/lvgl` sample, the on-screen button and the touch-screen inputs do not agree. Investigation shows that the orientation of the touch-input is using X and Y coordinates opposite of those used by the display.

This PR swaps the lvgl X/Y touch input coordinates for `lvgl_pointer` in the `m5stack_core2` board definition to match the orientation of the screen.

### CODE REVIEW

**Comment by kartben** [comment]
@mrmarteng

**Comment by MrMarteng** [comment]
> @MrMarteng

I (unfortunately) can confirm this issue. The fix resolves the issue.

I took the opportunity to extend the lvgl demo app by a drawing feature to test touch behavirour in more detail.
See https://github.com/zephyrproject-rtos/zephyr/pull/67083 for details. That could be helpful for testing. I will start the official PR once this fix is merged.

**Comment by MrMarteng** [comment]
Is this PR stuck? 
There is no reviewer defined yet.

**Comment by kartben** [comment]
> Is this PR stuck? There is no reviewer defined yet.

good catch. Looks like ESP32 based M5Stack boards are orphaned in the maintainers file, will get it fixed.

meanwhile, adding the ESP32 label...

**Review by faxe1008 — APPROVED** [review]
Didn't notice this PR, sorry for the wait. Thanks for submitting a fix 👍

**Review by sylvioalves — APPROVED** [review]

**Review by kartben — APPROVED** [review]

**Pull request merged**
